test-packages/type-tests/test/complex-types.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { and, asc, desc, or } from '@sap-cloud-sdk/core';
-import { TestComplexType, TestEntity } from '@sap-cloud-sdk/test-services/test-service';
+// import { TestComplexType, TestEntity } from '@sap-cloud-sdk/test-services/test-service';
+import { TestComplexType, TestEntity } from '../../test-services/srv/test-service';
```

Comment:
Is this intended?

Reply:
No, I thought this could make a difference but lerna puts a symbolic link so that this is equivalent.